extended-instructions.md

```diff
@@ -29,22 +29,12 @@ Step | Instruction | Screen Shot
 
 ### Step 4.3:
 
-1. Open the *index.html* file in Chrome. 
+To open the *index.html* file in Chrome. 
 
-**NOTE:** There are a few ways to open your assignment in Chrome:
 
-**Option 1: Open from the Finder**
-- Right click on your file in Atom
-- Select Reveal in Finder
-- Double click on the file in your finder window, to open in Chrome
-
-![screencast](./images/open-chrome-from-finder.gif)
```

Comment:
this screencast uses Atom. I'm happy giving students fewer options here, and biasing towards the terminal.